### DISCUSSION TITLE
Is this useful?

### DISCUSSION THREAD

**Discussion by schwer** [schwer/graphql-to-openapi#96] [Q&A]
Has anyone found a use for this codebase?

I initially did the work because I wanted to convert a large graphql system into a variety of openapi specifications that could be refactored into microservices.

Are there some other use cases that I'm not considering?

**Comment by wamdy** [comment]
目前我使用另一个工具：graphql-operation-generator，配合您的工作，可以比较完美的将graphql定义的文档转换为openapi规范，但是美中不足的一点是，生成的openapi文档没有定义可重用的公用模型或者说Components，我将尝试完善这一点，谢谢你的工作。

For now I use another tool: graphql-operation-generator, which works with you to convert the graphql-defined document to the openapi specification perfectly, but the fly in the ointment is that the resulting openapi document does not define a reusable common model or Composites, and I will try to improve this, many thanks for your work.
